src/components/o-cookie-disclaimer/js/disclaimer-handler.js

```diff
@@ -0,0 +1,25 @@
+const _global = window || global;
+
+class DisclaimerHandler {
+  constructor(wcNode) {
+    this.wcNode = wcNode;
+  }
+  init() {
+    this.wcNode.querySelector('.js-cookie-disclaimer__button').addEventListener('click', () => {
```

Comment:
Add event cleanup `removeEventListener`.

`on` returns an `off` function, which does this correctly to avoid memory leaks.
And all other js files implement a `destroy` function to deal with clean up